editoast/README.md

```diff
@@ -55,6 +56,12 @@ To install `rustfmt` and `clippy`, simply run:
 rustup component add rustfmt clippy
 ```
 
+To install `cargo-tomlfmt`, run (check `editoast/Dockerfile` for the exact SHA):
+
+```sh
+cargo install --locked --git https://github.com/tbrand/cargo-tomlfmt --rev 85354dfd771be75d7290435e613c07cb77b26e96
+```
+
```

Comment:
```suggestion
```

Reply:
Ooops, thanks for catching that. Fixed in [b9100c4](https://github.com/osrd-project/osrd/pull/7053/commits/b9100c473857a09778a67c789a60e9c6c5bb1a40).